zerver/lib/bugdown/__init__.py

```diff
@@ -665,7 +665,9 @@ def make_emoji(emoji_name, src, display_string):
     elt.set('src', src)
     elt.set('class', 'emoji')
     elt.set("alt", display_string)
-    elt.set("title", display_string)
+    # removes the colons at the ends. strips the underscore and joins with space instead.
+    title_string = " ".join(display_string[1:len(display_string) - 1].split("_"))
+    elt.set("title", title_string)
```

Comment:
this can just be `display_string[1:-1]`, and again use a replace function instead of a split and join. 
Also I'd just do it in one line, rather than defining a title_string variable